modules/mod_logging/templates/admin_log_ui.tpl

```diff
@@ -0,0 +1,73 @@
+{% extends "admin_log_base.tpl" %}
+
+{% block title %}{_ Log ui events _}{% endblock %}
+
+{% block title_log %}{_ Log user-interface events _}{% endblock %}
```

Comment:
Why do you use a hyphen in between user and interface here?

Reply:
Hmmm, I guess double-dutch?  better without?